### PR TITLE
docs: Sponsor Button

### PR DESCRIPTION
**Pull request by dmaynard51**
Added a sponsor button per the issue#98

### CODE REVIEW

**Comment by antfu** [comment]
Hey there😄! Thank you for interested in this project.

The screenshot in #98 is just an example from GitLens. Our donate information can be found [this file](https://github.com/breadsplit/breadsplit/blob/dev/meta/socials.ts). Would you like to update your PR?

Thanks.

**Comment by dmaynard51** [comment]
> @dmaynard51 Hey there😄! Thank you for interested in this project.
> 
> The screenshot in #98 is just an example from GitLens. Our donate information can be found [this file](https://github.com/breadsplit/breadsplit/blob/dev/meta/socials.ts). Would you like to update your PR?
> 
> Thanks.

Thanks Antfu, hopefully the update should be correct.

**Comment by antfu** [comment]
@dmaynard51 Thank you! But I am afraid the file format is not correct. You can look this as a reference.
https://github.com/eamodio/vscode-gitlens/blob/master/.github/FUNDING.yml

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # These are supported funding model platforms
 
-patreon: # https://www.patreon.com/user?u=19935929 (https://www.patreon.com/user?u=19935929)
-paypal: # https://www.paypal.me/breadsplit/USD5 (https://www.paypal.me/breadsplit/USD5)
+patreon: https://www.patreon.com/user?u=19935929
```
Comment (antfu) [inline .github/FUNDING.yml]: Should this be a username?

**Comment by antfu** [comment]
@dmaynard51 I am sorry, could you look through the document again? I can simply edit it rather than telling you how to correct it. But that make this PR meaningless.

**Comment by dmaynard51** [comment]
> @dmaynard51 I am sorry, could you look through the document again? I can simply edit it rather than telling you how to correct it. But that make this PR meaningless.

Sorry this is pretty much my first attempt at contributing. I read up a bit on the yml file, and saw fixed the paypal page to go under custom.

File: .github/FUNDING.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # These are supported funding model platforms
 
 patreon: https://www.patreon.com/breadsplit
```
Comment (antfu) [inline .github/FUNDING.yml]: Also the patreon field should be a usename alone instead of an url I suppose

**Comment by antfu** [comment]
@dmaynard51 Thanks

**Pull request merged**